app/static/tests/unit/components/sessions/sessionsPage.test.ts

```diff
@@ -376,14 +371,18 @@ describe("SessionsPage", () => {
 
     it("can save show unlabelled sessions preference", async () => {
         const wrapper = getWrapper({ sessionsMetadata }, currentSessionId);
-        await wrapper.find("input#show-unlabelled-check").trigger("click");
+        const checkbox = wrapper.find("input#show-unlabelled-check");
+        (checkbox.element as HTMLInputElement).checked = false;
+        await checkbox.trigger("change");
         expect(mockSaveUserPreferences).toHaveBeenCalledTimes(1);
         expect(mockSaveUserPreferences.mock.calls[0][1]).toStrictEqual({ showUnlabelledSessions: false });
     });
 
     it("can save show duplicate sessions preference", async () => {
         const wrapper = getWrapper({ sessionsMetadata }, currentSessionId);
-        await wrapper.find("input#show-duplicates-check").trigger("click");
+        const checkbox = wrapper.find("input#show-duplicates-check");
+        (checkbox.element as HTMLInputElement).checked = true;
+        await checkbox.trigger("change");
```

Comment:
ahh so this one was actually a bit weird, basically because we are using a `v-model` in the component and in the last PR the component i was testing was using an explicit event like `@input` or `@change`, `v-model` works in a slightly different way, it requires a `change` event to trigger